### PR TITLE
Parse translations files

### PR DESCRIPTION
**Pull request by NorseUA**

### CODE REVIEW

**Comment by PhilipKram** [comment]
@NorseUA Add unit tests

**Comment by NorseUA** [comment]
Setup of testing copied from Phoenix with small changes

File: lib/unzip/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const unzipTranslationsFromStream = (stream, callback) => stream
+  .pipe(unzip.Parse())
+  .on('entry', (entry) => {
+    const { type, path } = entry;
+    const translationsFileType = detectTranslationsFileType(path);
+
+    if (type === entryTypes.FILE && !!translationsFileType) {
+      entry.on('data', (data) => {
+        const fileText = data.toString();
+        textParser(fileText, translationsFileType)
```
Comment (PhilipKram) [inline lib/unzip/index.js]: unzip should contain only related thing to unzip
`textParser` is not part of unzip.
Event we would need to parser part as separate public methods (both files to data and data to files)

Comment (NorseUA) [reply lib/unzip/index.js]: This part not finished yet. It only for checking etc.

**Pull request merged**